_includes/transcription.html

```diff
@@ -0,0 +1,33 @@
+{%- assign id= include.id -%}
+{%- assign title= include.title -%}
+{%- assign alt= include.alt -%}
+
+<div class="fr-transcription" id="transcription-{{id}}">
+    <button class="fr-transcription__btn" aria-expanded="false" aria-controls="fr-transcription-collapse-transcription-{{id}}">Transcription</button>
+    <div class="fr-collapse" id="fr-transcription-collapse-transcription-{{id}}">
+        <div class="fr-transcription__footer">
+            <div class="fr-transcription__actions-group">
+                <button aria-controls="fr-transcription-modal-transcription-{{id}}" aria-label="Agrandir la transcription" data-fr-opened="false" class="fr-btn--fullscreen fr-btn">Agrandir</button>
+            </div>
+        </div>
```

Comment:
C'est un peu étrange d'avoir 2 boutons ici 🤔. Le problème principal c'est que du coup tu as deux points d'arrêt identiques en navigation au clavier. Tu peux pas avoir plutôt un truc du genre
```suggestion
<button class="fr-transcription__btn" aria-expanded="false" aria-controls="fr-transcription-collapse-transcription-{{id}}"><span>Transcription</span>
    <span class="fr-collapse" id="fr-transcription-collapse-transcription-{{id}}">
        <span class="fr-transcription__footer">
            <span class="fr-transcription__actions-group">
                <span class="fr-btn--fullscreen fr-btn">Agrandir</span>
            </span>
        </span>
    </span>
</button>
```

Reply:
J'ai repris [le code de la documentation](https://main--ds-gouv.netlify.app/example/component/transcription/) : 
1. Le premier bouton déroule l'accordéon avec la transcription
2. Le 2ème est dans l'accordéeon, et il permet d'afficher la transcription en grand écran.

Le second n'est donc pas accessible au clavier tant que le 1er n'a pas été actionné